src/components/sound-editor/sound-editor.jsx

```diff
@@ -52,6 +54,16 @@ const messages = defineMessages({
         description: 'Title of the button to save trimmed sound',
         defaultMessage: 'Save'
     },
+    undo: {
```

Comment:
We may want to share certain strings. However it also can be useful to keep the separate because even strings that are the same in english might be slightly different in other languages depending on context. ¯\_(ツ)_/¯